src/app/external-cluster-wizard/steps/external-cluster/provider/gke/component.ts

```diff
@@ -58,6 +61,11 @@ enum Controls {
   MinCount = 'minCount',
 }
 
+enum KubernetesVersionMode {
+  StaticVersion = 'Manual',
```

Comment:
can we rename it too?
```suggestion
  StaticVersion = 'static',
  ReleaseChannel = 'release',
```

ah! i guess we need to send to same value. You can skip it than